docs/source/docker/advanced/active_learning.rst

```diff
@@ -55,7 +55,7 @@ We can start a Lightly Worker using the following command
 
 .. code-block:: console
 
-  docker run --rm --gpus all -it \
+  docker run --shm-size="1024m" --rm --gpus all -it \
```

Comment:
Just FYI you can also use gigabytes as units: `--shm-size="1g"`

Reply:
I know. But I thought it's more handy to use m (for megabytes) as users can easily adapt to 512, 1500 etc.